index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/15d46053c4.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <title>Canine Conference</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <main>
+    <nav class="menucontainer">
+      <i class="fa-solid fa-xmark close-btn"></i>
+      <ul class="menuList">
+        <li class="menu-link"><a href="#intro">Home</a></li>
+        <li class="menu-link"><a href="#program">Main Program</a></li>
+        <li class="menu-link"><a href="#featuredSpeakers">Featured Speakers</a></li>
+        <li class="menu-link"><a href="about.html#info-2">About</a></li>
+        <li class="menu-link"><a href="#about-html#intro-2">Logo</a></li>
+        <li class="menu-link"><a href="about.html#allconcert">Past Conference</a></li>
+      </ul>
+    </nav>
+
+    <header class="menu">
+        <div class="desktopmenu">
+          <a><i class="fa-brands fa-facebook-f"></i></a>
+          <a><i class="fa-brands fa-twitter"></i></a>
+          <a>English</a>
+          <a>My Page</a>
+          <a id="menu-1">Logout</a>
+        </div>
+
+        <div class="logomenu">
+          <nav class="mainlogo">
+            <a href="#" class="mlogo1">DLGC</a>
+          </nav>
+          <nav class="menu-container">
+            <ul class="menulist">
+              <li class="menulink"><a href="about.html">About</a></li>
+              <li class="menulink"><a href="#program">Program</a></li>
+              <li class="menulink"><a href="#join">Join</a></li>
+              <li class="menulink"><a href="#sponsor">Sponsor</a></li>
+              <li class="menulink"><a href="#news">News</a></li>
+              <button class="campaignBtn">DLGC Campaign</button>
+            </ul>
+          </nav>
+        </div>        
+      
+      <div class="bg-image">
+        <nav class="menuBars">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </nav>
+      </div>
+    </header>
+  
+    <section class="intro">
+      <h1>"Hello! Dog Lovers"</h1>
```

Comment:
- [ ] This title is a subtiltle and as such it should be wrapped in a `hx` tag such as h2, h3 e.t.c.  This applies to the about page as well